dll/win32/shell32/shlfileop.cpp

```diff
@@ -663,6 +663,40 @@ EXTERN_C DWORD WINAPI Win32DeleteFileW(LPCWSTR path)
     return (SHNotifyDeleteFileW(NULL, path) == ERROR_SUCCESS);
 }
 
+/************************************************************************
+ * ShowErrorMessage          [internal]
+ *
+ * Show message box if operation failed
+ *
+ * PARAMS
+ *  op         [I]   File Operation context
+ *  error      [I]   Error code
+ *  src        [I]   Source file full name
+ *
+ */
+static void ShowErrorMessage(FILE_OPERATION *op, DWORD error, LPCWSTR src)
```

Comment:
Since you are already passing `FILE_OPERATION` into this function, it could be refactored even better. Here are my suggestions:
```suggestion
static DWORD CheckForError(FILE_OPERATION *op, DWORD error, LPCWSTR src)
```